src/OpenApi/perf/Microbenchmarks/OpenApiSchemaComparerBenchmark.cs

```diff
@@ -60,4 +60,10 @@ public void OpenApiSchema_GetHashCode()
     {
         OpenApiSchemaComparer.Instance.GetHashCode(_schema);
     }
+
+    [Benchmark]
+    public void OpenApiSchema_Clone()
+    {
+        _ = OpenApiSchemaExtensions.Clone(_schema);
```

Comment:
Rather than void, should these benchmark methods return the value from the benchmarked method? I'm sure I read somewhere once that's considered a good idea for BDN.

Reply:
I had trouble finding this guidance only but it seems sensible enough, we have other benchmarks that do it and it didn't appear to affect the results 🤷🏽‍♀️

Reply:
Had a quick look and found it here - [Avoid dead code elimination](https://benchmarkdotnet.org/articles/guides/good-practices.html#avoid-dead-code-elimination).

They also use the return values in their [Getting Started guide](https://benchmarkdotnet.org/articles/guides/getting-started.html#step-3-design-a-benchmark).

I guess it's a "always do it because it _might_ affect the benchmark" kinda piece of guidance.

Reply:
Excellent Google-fu 👏🏽